Tasks/NpmAuthenticate/make.json

```diff
@@ -0,0 +1,17 @@
+{
+    "common": [
+        {
+            "module": "../Common/npm-common",
+            "type": "node",
+            "compile": true
+        }
+    ],
+    "rm": [
+        {
+            "items": [
+                "node_modules/nuget-task-common/node_modules/vsts-task-lib"
```

Comment:
I don't think this is needed, we're not even including nuget-task-common in the npm tasks.

Reply:
Removed.